src/pages/index.js

```diff
@@ -70,5 +181,33 @@ const addCardFormValidator = new FormValidator(
   constants.addProfileModalForm
 );
 
+const avatarValidator = new FormValidator(
+  constants.settings,
+  constants.profileAvatarForm
+);
+
 addCardFormValidator.enableValidation();
 editCardFormValidator.enableValidation();
+avatarValidator.enableValidation();
+
+function likeCard(card) {
+  api
+    .likeCard(card.getId())
+    .then(() => {
+      card.setIsLiked();
+    })
+    .catch((err) => {
+      console.error(err);
+    });
+}
+
+function removeLike(card) {
+  api
+    .removeLikeCard(card.getId())
+    .then(() => {
+      card.setIsLiked();
```

Comment:
Needs correction: this method has one argument - the current likes state

Reply:
Corrected!